src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { app } from './app';
+import { log } from './logger/logger';
+
+const SERVER_PORT = process.env.PORT || 3000;
```

Comment:
Should be in the config because you want to parse it and validate that it is a valid port number